schema/private.openapi.yaml

```diff
@@ -315,6 +336,36 @@ components:
       - org_id
       - hosts
 
+    JobRequestBody:
+      type: object
+      properties:
+        org_id:
+          $ref: '#/components/schemas/OrgId'
+        principal:
+          $ref: '#/components/schemas/Principal'
+        url:
+          $ref: './public.openapi.yaml#/components/schemas/Url'
```

Comment:
In the design doc it mentions that this high level playbook dispatch endpoint will require an "prefix" that will be passed down to cloud connector:

> prefix to the playbook YAML generation endpoint for the corresponding service.  This prefix will be used to construct the request URL that the Recipient will send back to the client service in order to generate the correct playbook with the corresponding hosts.

I just wanted to make sure, that prefix is the same as the `url` that our users already send us in our lower level `/dispatch` endpoint, right?